addon/components/polaris-resource-list/filter-control/date-selector.js

```diff
@@ -128,9 +127,9 @@ export default Component.extend({
    * @default null
    * @private
    */
-  initialConsumerFilterKey: null,
+  initialConsumerFilterKey = null;
 
-  dateComparatorOptions: computed(function() {
+  get dateComparatorOptions() {
```

Comment:
Can this just be defined as a regular property now?

Reply:
yeap, I think we can....nice catch 👍

Reply:
🤦‍♂ 👍